QueueInteropTransport.php

```diff
@@ -77,13 +78,18 @@ public function get(): iterable
             throw $e;
         }
 
-        return array(
-            $this->serializer->decode(array(
-                'body' => $interopMessage->getBody(),
-                'headers' => $interopMessage->getHeaders(),
-                'properties' => $interopMessage->getProperties(),
-            )),
-        );
+        /** @var Envelope $envelope */
+        $envelope = $this->serializer->decode(array(
+            'body' => $interopMessage->getBody(),
+            'headers' => $interopMessage->getHeaders(),
+            'properties' => $interopMessage->getProperties(),
+        ));
+
+        if ($envelope) {
```

Comment:
The `->decode()` method *always* returns an `Envelope` - I don't think there's a reason to heck for that here.